src/util.ts

```diff
@@ -478,3 +524,50 @@ export function getDebugOption(): boolean {
 
   return false
 }
+
+export function writeToFile(data: string, filePathOverride?: string): string {
+  const filePath = filePathOverride || path.join(os.tmpdir(), `${cuid()}.png`)
+  fs.writeFileSync(filePath, Buffer.from(data, 'base64'))
+  return filePath
+}
+
+function getS3BucketName() {
+  return process.env['CHROMELESS_S3_BUCKET_NAME']
+}
+
+function getS3BucketURL() {
```

Comment:
It's nit-picky, but I'd prefer we call this `getS3BucketUrl()` as it's more consistent with the naming convention throughout the rest of the code.